src/esp/gfx/Simulator.h

```diff
@@ -61,8 +61,10 @@ class Simulator {
 
   void saveFrame(const std::string& filename);
 
+  RenderingTarget::ptr createRenderingTarget(int height, int width);
```

Comment:
I am not a broken radio, but I still would like to suggest using the convention W x H so that it is uniform everywhere in the C++ backend (including Magnum), which is less error-prone in future development.